### PR TITLE
Debug: idempotent transformation

### PR DESCRIPTION
**Pull request by bryanlatten**
- Also added optimized autoloader for composer

### CODE REVIEW

File: container/root/run.d/01-install.sh [hunk 1]
```diff
@@ -6,6 +6,6 @@
 if [[ -f /root/.composer/config.json ]]
 then
   echo "[install] app dependencies"
-  composer install
+  composer install --optimize-autoloader
```
Comment (bryanlatten) [inline container/root/run.d/01-install.sh:9]: missing from earlier! should give us a small performance boost

File: container/root/etc/php5/mods-available/apcu.ini [hunk 2]
```diff
@@ -0,0 +1,2 @@
+extension=apcu.so
+apc.enable_cli=1
```
Comment (bryanlatten) [inline container/root/etc/php5/mods-available/apcu.ini]: allows apc memory to be changed/altered during CLI runs

File: Dockerfile [hunk 3]
```diff
@@ -18,8 +18,8 @@ RUN locale-gen en_US.UTF-8 && export LANG=en_US.UTF-8 && \
 # Update package cache with new PPA, install language and modules
 RUN apt-get update && \
     apt-get -yq install \
-        php5=5.6.12+dfsg-1+deb.sury.org~trusty+1 \
-        php5-fpm=5.6.12+dfsg-1+deb.sury.org~trusty+1 \
+        php5=5.6.13+dfsg-1+deb.sury.org~trusty+1 \
+        php5-fpm=5.6.13+dfsg-1+deb.sury.org~trusty+1 \
```
Comment (bryanlatten) [inline Dockerfile:22]: might as well up the point version

**Comment by ingluisjimenez** [comment]
:+1:

**Comment by chrishenry** [comment]
:+1:

**Pull request merged**